### DISCUSSION TITLE
Information about stable  version

### DISCUSSION THREAD

**Discussion by Vinayakmane47** [ckan/ckan#7173] [General]
I wanted to install stable version of CKAN , please let me know the stable one

**Comment by kowh-ai** [comment]
Both these CKAN versions are the latest stable versions: 
1. **2.9.7** 
2. **2.8.12**

**Reply by Vinayakmane47** [reply]
i am getting this error while installing on windows 10 pro

ckan | exec /ckan-entrypoint.sh: no such file or directory

**Comment by Vinayakmane47** [comment]
i am getting this error while installing on windows 10 pro 

ckan | exec /ckan-entrypoint.sh: no such file or directory
